CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ ENHANCEMENTS:
 * Allow test run blocks to reference previous run block's module outputs ([#1129](https://github.com/opentofu/opentofu/pull/1129))
 
 BUG FIXES:
+* Fix hook test flakiness by deterministically waiting for heartbeats to execute ([$1153](https://github.com/opentofu/opentofu/issues/1153))
```

Comment:
This might get confused with the `tofu test` feature.  I'm not opposed to having this in the changelog, but we probably want to be careful about wording.

Reply:
Yeah, I'd propose changing test to unit test.

Reply:
Might also make sense that it's referring to view hooks, so `Fix view hooks unit test flakiness...`

Reply:
Sounds good, updated